src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {
```

Comment:
should be an abstract class not interface
`public abstract class Figure implements AreaCalculator, Drawable {`